spec/models/form/sales/questions/discounted_ownership_type_spec.rb

```diff
@@ -0,0 +1,45 @@
+require "rails_helper"
+
+RSpec.describe Form::Sales::Questions::DiscountedOwnershipType, type: :model do
+  subject(:question) { described_class.new(question_id, question_definition, page) }
+
+  let(:question_id) { nil }
+  let(:question_definition) { nil }
+  let(:page) { instance_double(Form::Page) }
+
+  it "has correct page" do
+    expect(question.page).to eq(page)
+  end
+
+  it "has the correct id" do
+    expect(question.id).to eq("type")
+  end
+
+  it "has the correct header" do
+    expect(question.header).to eq("What is the type of discounted ownership sale?")
+  end
+
+  it "has the correct check_answer_label" do
+    expect(question.check_answer_label).to eq("Type of discounted ownership sale")
+  end
+
+  it "has the correct type" do
+    expect(question.type).to eq("radio")
+  end
+
+  it "is not marked as derived" do
+    expect(question.derived?).to be false
+  end
+
+  it "has the correct answer_options" do
+    expect(question.answer_options).to eq({
+      "8" => { "value" => "Right to Aquire (RTA)" },
+      "14" => { "value" => "Preserved Right to Buy (PRTB)" },
+      "27" => { "value" => "Voluntary Right to Buy (VRTB)" },
+      "9" => { "value" => "Right to Buy (RTB)" },
+      "29" => { "value" => "Rent to Buy - Full Ownership" },
+      "21" => { "value" => "Social HomeBuy for outright purchase" },
+      "22" => { "value" => "Any other equity loan scheme" },
+    })
+  end
+end
```

Comment:
I have mixed feelings on the usefulness of tests for the questions (at least when they're this simple). It feels very repetitious, but I worry extending that thought just amounts to arguing for them being data rather than code again 😅

Reply:
It is testing a lot of content, but I suppose that's because the content impacts the behaviour. I think these tests give me a little bit more confidence of the codebase, but deffo open to alternative ways of testing this 😅

Reply:
Definitely a good point on the mappings. And yeah, for the content I don't have a better option off the top of my head, it might just be that on balance this is the best bet.